tests/e2e/workloads/bdi/test_bdi_workload.py

```diff
@@ -0,0 +1,249 @@
+from datetime import datetime
+import logging
+import pytest
+from ocs_ci.framework.testlib import E2ETest, workloads
+from tests.e2e.workloads.bdi.fixtures import (
+    clone_ibm_chart_project,
+    create_machineset,
+    install_helm,
+    create_db2u_project,
+    create_security_context_constraints,
+    security_prereqs,
+    create_secretes,
+    ibm_container_registry,
+)
+from tests.e2e.workloads.bdi.helper import (
+    init_configure_workload_yaml,
+    init_data_load_yaml,
+    init_run_workload_yaml,
+    install_db2u,
+    clean_temp_files,
+)
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs import ocp
+
+import tempfile
+from ocs_ci.utility import templating
+
+log = logging.getLogger(__name__)
+
+
+@workloads
+@tier1
+@pytest.mark.usefixtures(
+    clone_ibm_chart_project.__name__,
+    create_machineset.__name__,
+    install_helm.__name__,
+    create_db2u_project.__name__,
+    create_security_context_constraints.__name__,
+    security_prereqs.__name__,
+    create_secretes.__name__,
+    ibm_container_registry.__name__,
```

Comment:
Do you think it will be a good idea to create a BDI class and convert these functions as class functions?
WDYT?

If yes, This can be taken as a follow up enhancement PR.

Reply:
IMO, we should re-assess the actual usage for this tool.
If we want to run multiple TCs one-by-one, we need to change the fixtures to be in module/class/function scopes.
But anyway, I don't think we should change it until we decide.

Reply:
What is the benefit of having those functions as class methods?